package.json

```diff
@@ -23,7 +23,7 @@
     "rebuild": "npm run clean && npm run build",
     "clean": "lb-clean dist *.tsbuildinfo .eslintcache",
     "start": "npm run rebuild && node dist/server",
-    "server": "node dist/server",
+    "server": "nodemon dist/server",
```

Comment:
Maybe have `"server:dev": "nodemon dist/server"`?

Reply:
I think the `start` script is doing a production spinup while `server` is used for dev